### PR TITLE
Feature: Billing api + profile

### PR DESCRIPTION
**Pull request by ksol**
The inline doc could has room for improvement; it'll have to wait until the proper documentation is done.

### CODE REVIEW

File: src/Billing/index.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+  /**
+   * Return the billing profile of current user
+   * @see https://developers.scalingo.com/billing#billing-profile
+   * @return {Promise<BillingProfile | APIError>}
+   */
+  profile() {
+    return unpackData(
+      this._client.billingApiClient().get('/profile'),
+      'profile',
```
Comment (EtienneM) [inline src/Billing/index.js]:
```suggestion
      'billing_profile',
```
The dev doc says that the key of the response is `billing_profile`

File: src/Billing/index.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+  /**
+   * Create the billing profile of current user. Returns an error if already existing
+   * @see https://developers.scalingo.com/billing#billing-profile
+   * @param {BillingProfile} profile The billing profile to create
+   * @return {Promise<BillingProfile | APIError>}
+   */
+  createProfile(profile) {
+    return unpackData(
+      this._client.billingApiClient().post('/profiles', { profile }),
```
Comment (EtienneM) [inline src/Billing/index.js]: Are you sure it's plural? `profiles`?

File: src/Billing/index.js (hunk 1 above)
Comment (ksol) [reply src/Billing/index.js]: The dev docs refer to the api.scalingo endpoint, not the billing api's ones; those are different and not documented yet. I've relied on having access to the source of the api to find the appropriate interface.

File: src/Billing/index.js (hunk 2 above)
Comment (ksol) [reply src/Billing/index.js]: Same comment as above; also, the examples I've added are running as expected

File: src/Billing/index.js (hunk 1 above)
Comment (EtienneM) [reply src/Billing/index.js]: Arf OK, sorry

**Pull request merged**